src/history/hs_rec.c

```diff
@@ -69,21 +68,17 @@ __hs_insert_record(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_BTREE *btree,
   const uint8_t type, const WT_ITEM *hs_value, WT_TIME_WINDOW *tw, bool error_on_ts_ordering)
 {
     WT_CURSOR_BTREE *hs_cbt;
-    WT_DECL_ITEM(hs_key);
-#ifdef HAVE_DIAGNOSTIC
     WT_DECL_ITEM(existing_val);
-#endif
+    WT_DECL_ITEM(hs_key);
```

Comment:
Juuust in case can you check we've run s-all here? The order of variable declarations looks a bit different to what I expected.

Reply:
Re-ran s_all, did not seem to change anything: [ea2caa5](https://github.com/wiredtiger/wiredtiger/pull/8606/commits/ea2caa5383d6da4a6a9580b41f6fe00365e5b088)